sunpy/timeseries/sources/goes.py

```diff
@@ -285,10 +291,13 @@ def _parse_netcdf(filepath):
             else:
                 raise ValueError(f"The file {filepath} doesn't seem to be a GOES netcdf file.")
 
-        data = DataFrame({"xrsa": xrsa, "xrsb": xrsb}, index=times.datetime)
+        data = DataFrame({"xrsa": xrsa, "xrsb": xrsb, "xrsa_quality": xrsa_quality,
+                         "xrsb_quality": xrsb_quality}, index=times.datetime)
         data = data.replace(-9999, np.nan)
         units = OrderedDict([("xrsa", u.W/u.m**2),
-                             ("xrsb", u.W/u.m**2)])
+                             ("xrsb", u.W/u.m**2),
+                             ("xrsa_quality", int),
```

Comment:
Is it better to name this xrsa_flags as well?

Reply:
the value is a flag based on the following mask:

```
[0,   1,   2,   4,   8,  16,  32,  64, 128, 256, 512]
[good_data, eclipse, particle_spike, calibration, off_point, temperature_error, data_quality_error, pointing_error, invalid_mode, missing_data, L0_error]
```

any value different from 0 means something is not correct

e.g. 72 = 64 + 8 = pointing_error + off_point